book/09-git-and-other-scms/sections/import-bzr.asc

```diff
@@ -129,12 +129,15 @@ $ git reset --hard HEAD
 ===== Ignoring the files that were ignored with .bzrignore
 
 Now let's have a look at the files to ignore.
-As `.bzrignore`'s format is completely compatible with `.gitignore`'s format, the simplest is to rename your `.bzrignore` file.
-You will also have to create a commit that contains this change for the migration:
+The first thing to do is to rename `.bzrignore` into `.gitignore`.
+Moreover, if the `.bzrignore` file contains one or several lines starting with "!!" or "RE:", you'll have to modify it and perhaps create several `.gitignore` files in order to ignore exactly the same files that `.bzrignore` allowed.
```

Comment:
Let's remove the "moreover," it doesn't seem to work here. Also, let's end this sentence like so:

> … in order to ignore exactly the same files that Bazaar was ignoring.